eclair-core/src/main/resources/reference.conf

```diff
@@ -64,6 +64,7 @@ eclair {
   channel-flags = 1 // announce channels
 
   dust-limit-satoshis = 546
+  max-remote-dust-limit-satoshis = 1000
```

Comment:
Should we even set this to 546? I think no-one ever used something else.

Reply:
If we can't see any other value on our logs in the past month or so I guess yes.

Reply:
Actually it looks like we have some people using `573 sat`. I'll change this limit to `600 sat` then.